docs/team/rohansapra.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: page
+title: Rohan Sapra's Project Portfolio Page
+---
+
+### Project: InsureIQ
+
+InsureIQ is for car insurance agents who type fast over CLI and have to keep track of multiple customers' vehicles with the car insurance policies.
+The user interacts with it using a CLI, and it has a GUI created with JavaFX.
+
+Given below are my contributions to the project.
+
+* **New Feature**:
+    * to be added soon
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=suryanshkushwaha&breakdown=true)
```

Comment:
Should change this to your own reposense link

Reply:
Done